src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageEndpoint.cs

```diff
@@ -14,12 +15,14 @@
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.Extensions.Logging;
 using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using Range = OmniSharp.Extensions.LanguageServer.Protocol.Models.Range;
```

Comment:
Does this actually remove the ambiguity? Wouldn't you have to name it like `OmniSharpRange`?

Reply:
Removes ambiguity for this file 😄